query/src/main/scala/filodb/query/exec/AggrOverRangeVectors.scala

```diff
@@ -132,7 +134,7 @@ object RangeVectorAggregator extends StrictLogging {
       // now reduce each group and create one result range vector per group
       val groupedResult = mapReduceInternal(rvs, rowAgg, skipMapPhase, grouping)
       groupedResult.map { case (rvk, aggHolder) =>
-        val rowIterator = aggHolder.map(_.toRowReader)
+        val rowIterator = new CustomCloseCursor(aggHolder.map(_.toRowReader))(rvs.foreach(_.rows().close()))
```

Comment:
seems like we are invoking close redundantly for each group?

Reply:
Good Point. I'll try to address.

Reply:
Addressed. Thanks for catching!